build.sh

```diff
@@ -88,6 +92,8 @@ if [ "$1" == "-h"  -o  "$1" == "--help" ]; then
     echo build.sh -i/--install: Install Kmesh.
     echo build.sh -c/--clean: Clean the built binary.
     echo build.sh -u/--uninstall: Uninstall Kmesh.
+    echo Environment Variables:
+    echo SKIP_DEPENDENCIES=1         : Skip dependency installation.
```

Comment:
We usually use `make build` or `make docker` to compile Kmesh. Can make this env take effect under such conditions?

Reply:
make build and make docker are run in container, need to make sure the dependencies are all installed in kmesh-build image.

Reply:
Need a way to pass the env to the building container